tests/foreman/cli/test_docker.py

```diff
@@ -138,7 +138,7 @@ def test_create_one_docker_repo(self):
 
     @tier1
     @run_only_on('sat')
-    def test_create_multiple_docker_repo(self):
+    def test_positive_create_multiple_docker_repos_using_same_product(self):
```

Comment:
I want to stress the fact that it is using the product.  So I propose `test_positive_create_repos_using_same_product`